# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/gettext-redefinition.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./gettext-runtime/intl/eval-plural.h.orig	2005-05-20 23:05:58.000000000 +0200
+++ ./gettext-runtime/intl/eval-plural.h	2006-09-02 15:19:32.000000000 +0200
@@ -23,7 +23,7 @@
 /* Evaluate the plural expression and return an index value.  */
 STATIC
 unsigned long int
-internal_function
+/*internal_function*/
 plural_eval (struct expression *pexp, unsigned long int n)
 {
   switch (pexp->nargs)
